libsolidity/codegen/ir/Common.h

```diff
@@ -22,11 +22,29 @@
 
 #include <libsolidity/ast/AST.h>
 
+#include <algorithm>
 #include <string>
 
 namespace solidity::frontend
 {
 
+/**
+ * Structure that describes arity and co-arity of a function, i.e. the number of its inputs and outputs.
```

Comment:
Maybe clarify: Arity of the yul function representing an internal call to a Solidity function.

Reply:
I'll rename it to `YulArity`. It could actually be used for both Solidity and Yul arity but I guess, if we ever have a need for something like that for Solidity functions it might be better to have separate types.

Reply:
Done.